src/common/meta/src/reconciliation/reconcile_database.rs

```diff
@@ -45,6 +45,8 @@ use crate::reconciliation::reconcile_database::utils::wait_for_inflight_subproce
 use crate::reconciliation::reconcile_table::resolve_column_metadata::ResolveStrategy;
 use crate::reconciliation::utils::Context;
 
+pub(crate) const DEFAULT_PARALLELISM: usize = 64;
```

Comment:
Can this configuration be set by the admin function arguments?

Reply:
I'll introduce it in a separate PR.